src/lib.rs

```diff
@@ -888,6 +888,20 @@ impl<T> Vob<T> {
         &mut self.vec
     }
 
+    /// Get a reference to the underlying data structure
+    ///
+    /// `Vob` stores the bits in little-endian order. Any beyond `self.len()` should be `0`.
```

Comment:
At the risk of sounding like a lawyer, in retrospect this should say "will" rather than "should". Similarly we should update the comment "Any bits past `self.len()` should be set to 0 in the last block." above to saying "must be set to 0".